content/docs/user-guide/how-to/add-output-to-stage.md

```diff
@@ -0,0 +1,46 @@
+# Add Output to Stage
+
+There are situations where we have executed a stage (either by writing
+`dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later notice
+that output(s) already existing in workspace are missing from `dvc.yaml`.
+Re-executing a stage to specify all the outputs is an expensive step, as it
+requires time and resources. Follow the steps below to add existing output(s) to
+a stage without executing it again.
```

Comment:
This is pretty good writing, nice. But reading it now I just have one last concern. Will people understand what we mean by "output(s) already existing in workspace"? ("THE workspace" BTW 😕)
We could clarify what an output is in this case: file(s) written by the stage command (which we want to track with DVC).

Reply:
Also, please use `<abbr>` arround "output" in the 2nd or last instance of the word (once we're clearly referring to outputs tracked by DVC).

Reply:
> add existing output(s) to a stage

Hmmm. How about something like "add existing files or directories as outputs to a stage (`outs` field)" to make sure we separate the concepts of files in the workspace from tracked outputs, like I mentioned earlier? Please rewrite the p along these lines. Almost there...

Reply:
I see you added a definition of what outputs are. That tries to address this but isn't subtle enough. I've finished this change for you as it's a little complicated I suppose, resolving here!

Reply:
Thank You. I tried to include that but the sentence became very complex so I thought I'd rather make a separate one. Thanks again btw.